testing/mopidy/mopidy.post-install

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+mkdir /etc/mopidy/extensions.d
+mkdir /var/cache/mopidy
+mkdir /var/log/mopidy
+mkdir /var/lib/mopidy/local
+mkdir /var/lib/mopidy/media
+mkdir /var/lib/mopidy/playlists
+
+chown -R mopidy:audio /etc/mopidy/extensions.d
+chown -R mopidy:audio /var/cache/mopidy
+chown -R mopidy:audio /var/lib/mopidy
```

Comment:
I think those should be created in APKBUILD in `package()`.

Reply:
But they have to be owned by the correct user and group. Since the user is created in the pre-install hook, I don't think I can add the correct owner from `package()` right? Would the correct fix be creating the directories in `package()`, and then chowning it in the post-install hook?